pyproject.toml

```diff
@@ -1,3 +1,35 @@
+[build-system]
+requires = ["flit_core >=3.2,<4"]
+build-backend = "flit_core.buildapi"
+
+[project]
+name = "pip_api"
+version = "0.0.34"
+description = "An unofficial, importable pip API"
+readme = "README.md"
+license = { file = "LICENSE" }
+authors = [{ name = "Dustin Ingram", email = "di@python.org" }]
+classifiers = [
+  "License :: OSI Approved :: Apache Software License",
+  "Programming Language :: Python :: 3 :: Only",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
+]
+dependencies = ["pip"]
+requires-python = ">=3.8"
+
+[project.urls]
+Homepage = "https://pypi.org/project/pip-api/"
+Issues = "https://github.com/di/pip-api/issues"
+Source = "https://github.com/di/pip-api"
+
+[tool.flit.sdist]
+include = ["tests/", "tox.ini"]
```

Comment:
NB: This is for consistency with the old sdists: the sdists currently include tests + tox configs, which is why the CI matrix works as-is (since the `$GITHUB_WORKSPACE` is expanded directly from the sdist, not a checkout).

Reply:
(Alternatively, we could remove these from the sdist entirely by changing the CI's behavior to not rely on their presence. But this was the path of least resistance 🙂)